test/unity/toolkits/drawing_classifier/test_dc_serialization.cxx

```diff
@@ -176,8 +185,211 @@ BOOST_AUTO_TEST_CASE(test_export_coreml) {
                    target + "Probability");
 }
 
+BOOST_AUTO_TEST_CASE(test_load_version) {
```

Comment:
In what way does this test `load_version`?

Reply:
I should remove this.